### ISSUE TITLE
Add a way to hide certain groups

### ISSUE THREAD

**Issue by tobyzerner** [flarum/core#845]
https://discuss.flarum.org/d/2181-special-users-profile-icons

**Comment by Averell7** [comment]
The CSS solution proposed in this discussion works very well : 

```
.App--discussion .Badge--group--Admin {
    display: none;
}
```
It is flexible (choose which group to hide) and works also with accentuated characters in group names.

**Comment by Luceos** [comment]
It does not hide it completely @Averell7 ; you can still inspect the html to find hidden badges. What @tobscure proposes is to add a property to a badge to allow it to be not parsed into html at all.

**Comment by luceos** [comment]
Keep open please

**Comment by franzliedke** [comment]
Implemented in #2129 / 6e8884f190395b5a06fb8ebee4f8bb4e3cb8c467. We're done here! :clap: :clinking_glasses:
